org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/util/JsonUtilTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2019 Sreekanth Nelakurthi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.aposin.licensescout.util;
+
+import org.aposin.licensescout.archive.ArchiveIdentifierVersion;
+import org.aposin.licensescout.model.Author;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+
+public class JsonUtilTest {
+
+    @Test
+    public void getNPMArchiveDescriptionTest() throws Exception {
+        File packageFile = new File(getClass().getClassLoader().getResource("package.json").getFile());
```

Comment:
Could you maybe extract this pattern used in every test to a `getResource(String)` method to keep the test more readable?

Reply:
Done

Reply:
👍